examples/tpcds-example.conf

```diff
@@ -0,0 +1,148 @@
+spark-bench = {
+  spark-submit-config = [{
+    spark-args = {
+      executor-memory = "512M"
+      num-executors = "8"
+//      executor-memory = "1G"
+//      num-executors = "195"
+    }
+    conf = {
+//      "spark.driver.extraJavaOptions" = "-Xrunjdwp:transport=dt_socket,server=n,address=localhost:5005,suspend=y,onuncaught=n"
+      "spark.ui.showConsoleProgress" = "false"
+      "spark.ui.enabled" = "false"
+      "spark.ui.retainedJobs" = "0"
+      "spark.ui.retainedStages" = "0"
+      "spark.ui.retainedTasks" = "0"
+      "spark.sql.ui.retainedExecutions" = "0"
+      "spark.sql.autoBroadcastJoinThreshold" = "-1"
+      "spark.sql.crossJoin.enabled" = "true"
+      "spark.serializer" = "org.apache.spark.serializer.KryoSerializer"
+      "spark.dynamicAllocation.enabled" = "false"
+      "spark.shuffle.service.enabled" = "false"
+      "spark.debug.maxToStringFields" = "100"
+      "spark.sql.warehouse.dir" = "hdfs:///tpcds-warehouse"
+//      "spark.sql.warehouse.dir" = "hdfs:///user/notebook/tpcds-warehouse"
+    }
+    suites-parallel = false
+    workload-suites = [
+      {
+        descr = "Generate a dataset"
+        benchmark-output = "console"
+        parallel = false
+        workloads = [
+//          {
```

Comment:
can we get rid of commented lines

Reply:
oops. those should actually be uncommented. was in a testing state where i didn't want to generated the data or queries.